e2e/cypress/integration/enterprise/cloud/billing/company_information_free_trial_spec.js

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+import {getRandomLetter} from '../../../../utils/index';
+
+describe('System Console - Company Information section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+
+        gotoCompanyInformationScreen();
```

Comment:
Same comment as the other PR - https://github.com/mattermost/mattermost-webapp/pull/8361/files#r688199955
```suggestion
        // # Visit Company Information page
        cy.visit('/admin_console/billing/company_info');
        cy.get('.admin-console__header').should('be.visible').and('have.text', 'Company Information');
```